src/db.py

```diff
@@ -241,14 +241,16 @@ def delete_list_by_id(list_id):
     my_db.close()
 
 
-def get_items_in_list(list_id):
+def get_items_in_list(list_uuid):
     my_db = cnxpool.get_connection()
     cursor = my_db.cursor()
-    query = "SELECT _id, Elements_Name, Elements_Value, Elements_User_id, Elements_Quantity FROM Tbl_Elements " \
-            "WHERE Elements_id = %s"
-    cursor.execute(query, (list_id,))
+    query = "SELECT Elements_Name, Elements_Value, Elements_Quantity FROM Tbl_Elements " \
+            "INNER JOIN Tbl_Lists TL on Tbl_Elements.List_id = TL._id " \
+            "WHERE TL.Lists_Uuid = %s"
+
+    cursor.execute(query, (list_uuid,))
     retVal = cursor.fetchall()
-    my_db.commit()
+
     cursor.close()
     my_db.close()
     return retVal
```

Comment:
The return value in this scenario will be a list of tuples. There wont be any mappings to parse the data. Could you please instead turn this into a list of dictionary/json objects being returned.

Reply:
changed to return dictionary